code/headers/st7735_buffered.hpp

```diff
@@ -21,28 +30,48 @@ namespace r2d2::display {
          * @param reset
          */
         st7735_buffered_c(hwlib::spi_bus &bus, hwlib::pin_out &cs,
-                          hwlib::pin_out &dc, hwlib::pin_out &reset);
+                          hwlib::pin_out &dc, hwlib::pin_out &reset)
+            : st7735_c<Cursor_Count, Display_Size_Width, Display_Size_Height>(bus, cs, dc, reset) 
+        {}
 
         /**
          * @brief Directly write a pixel to the screen
          *
          * @param x
          * @param y
-         * @param data
+         * @param data Data is the color of the pixel
          */
-        void set_pixel(uint16_t x, uint16_t y, const uint16_t data) override;
+        void set_pixel(uint16_t x, uint16_t y, const uint16_t data) override  {
+
+            const uint16_t t = __REV16(data); // make a copy and reverse byte order
+
+            // write pixel data to the buffer
+            this->buffer[x + (y * this->width)] = t;
+        }
 
         /**
-         * @brief Directly write multiple pixels to the screen
-         *
+         * @brief Directly write multiple pixels to the screen. This effectively
+         * draws a rectangle based on given location and size.
+         * 
+         * 
```

Comment:
empty newlines?

Reply:
What is wrong with this?

Reply:
Same as other newline comment

Reply:
fixed in c2dfd0d4e42b03c5f9eeb9234d5f7f92fd062ad3